packages/widgets/src/menubar.ts

```diff
@@ -403,8 +457,72 @@ export class MenuBar extends Widget {
           this.activeIndex = i;
         }
       });
+      // Calculate size of current menu
+      totalMenuSize += this._menuItemSizes[i];
+    }
+    // Render overflow menu if needed
+    if (this._overflowIndex > -1) {
+      // Create overflow menu
+      if (this._overflowMenu === null) {
+        this._overflowMenu = new Menu({ commands: new CommandRegistry() });
+        this._overflowMenu.title.label = '...';
+        this._overflowMenu.title.mnemonic = 0;
```

Comment:
Maybe worth adding a new optional method in `IRenderer`, e.g. `createOverflowMenu()`  that would allow to customise this? In particular downstream may want to customize label. You could provide a default implementation in `Renderer`.

Reply:
I see that 4e8c6253f1df8b5b5cf7ca10d699af082673dd99 added `renderOverflowItem` which is a copy-paste of `renderItem` and is used to replace it; I think that we should . This might be useful (though I would avoid copy-pasting the code and instead just invoke the other method by default), but this is not what I had in mind.

I meant to suggest that we should wrap rendering of the overflow menu so that the `...`. and mnemonic can be customized downstream. The lines that I commented on would then become for example:

```suggestion
        this.renderer.decorateOverflowMenu(this._overflowMenu.title)
```

This is very minor, I don't think we have to do this. To limit the surface of API changes I would also suggest reverting `renderOverflowItem` for now - if this is needed in the future we can always add it (removing things is more difficult).